client/src/scene/SceneGameArena.ts

```diff
@@ -177,6 +189,28 @@ export class SceneGameArena extends Phaser.Scene {
                 this.gameState.board,
                 Tetromino.rotateCW
             );
+        } else if (this.keys.shift.isDown) {
```

Comment:
Can you add in a note at the top of this function with like `FIXME: Spectators are able to press these buttons`